File: TwitterTesto/TwitterTesto/Tweet.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Net;
using System.IO;
using Newtonsoft.Json;

namespace TwitterTesto
{
    class Tweet : ContentPiece
    {
        string testOutput = "NOTHING";

        public void printHits()
        {
            var oAuthTwitterWrapper = new OAuthTwitterWrapper.OAuthTwitterWrapper();
			Console.Write("**** Time Line *****\n");
            testOutput = oAuthTwitterWrapper.GetMyTimeline();
            Console.Write(testOutput + "\n\n");
  
			Console.Write("**** Search *****\n");
			Console.Write(oAuthTwitterWrapper.GetSearch() + "\n\n");
			//oAuthTwitterWrapper.TimeLineSettings = new TimeLineSettings {
            Console.ReadLine();

        }

        public string getOutput()
        {
            return testOutput;
        }

        protected DateTime timeStamp(){
            return new DateTime(); 
        }

        public override void Display(){
            
        }

        public static int CompareByDate(ContentPiece piece1, ContentPiece piece2)
        {
            return piece1.timeStamp.CompareTo(piece2.timeStamp);
        }
    }
}

File: TwitterReddit/app/src/main/java/com/example/twitterreddit/Tweet.java
package com.example.twitterreddit;

import java.time.LocalDateTime;

class Tweet extends ContentPiece
        {
        String testOutput = "NOTHING";

public void printHits()
        {
        OAuthTwitterWrapper oAuthTwitterWrapper = new OAuthTwitterWrapper.OAuthTwitterWrapper();
        System.out.println("**** Time Line *****\n");
        testOutput = oAuthTwitterWrapper.GetMyTimeline();
            System.out.println(testOutput + "\n\n");

            System.out.println("**** Search *****\n");
            System.out.println(oAuthTwitterWrapper.GetSearch() + "\n\n");
        }

public String getOutput()
        {
        return testOutput;
        }


public void Display(){

        }

public static int compareTo(ContentPiece piece1, ContentPiece piece2)
        {
        return piece1.timeStamp.compareTo(piece2.timeStamp);
        }
        }
